utils/SystemSetup.ts

```diff
@@ -113,6 +114,15 @@ export async function setUpFunctionalSystem(
     usdInitialWbtcPrice = await getOnChainWbtcUsdPrice(hre);
   }
 
+  if (hre.network.name == "mainnetFork") {
```

Comment:
I wasn't sure if we wanted to add some generic, mandatory support for other chains in the future, so I decided to go with YAGNI and added it only to local deployment to test FE. WDYT?

Reply:
I don't think it's something we want to deal with, atm